WcaOnRails/db/migrate/20161117085757_cleanup_events_issue96.rb

```diff
@@ -0,0 +1,12 @@
+class CleanupEventsIssue96 < ActiveRecord::Migration
+  def up
+    remove_foreign_key :preferred_formats, column: :event_id
+    remove_foreign_key :competition_events, column: :event_id
```

Comment:
The point of this change is to remove events that were never official. The way this migration does this is by deleting all events and then adding in only the ones we want.

How would you guys write this migration?